### PR TITLE
Prepare 0.0.7

### PR DESCRIPTION
**Pull request by maurolepore**
Relates to https://github.com/2DegreesInvesting/r2dii.match/issues/324

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
 
-* Avoid failures due to changes in r2dii.data that are expected.
+* `match_names()` gains `...` to pass additional arguments to
+  `stringdist::stringsim()` (@evgeniadimi #310).
 
 # r2dii.match 0.0.6
 
-* Use snapshots via testthat 3e.
-* `match_names()` gains `...` to pass additional arguments to
-  `stringdist::stringsim()` (@evgeniadimi #310).
```
Comment (maurolepore) [inline NEWS.md:9]: These items were under the wrong heading.

File: NEWS.md [hunk 2]
```diff
@@ -1,12 +1,10 @@
 # r2dii.match (development version)
 
-* Avoid failures due to changes in r2dii.data that are expected.
```
Comment (maurolepore) [inline NEWS.md:3]: NEWS.md focuses on user-facing changes only.

**Review by maurolepore — COMMENTED** [review]
A bunch of changes are just the output of revdepcheck

**Review by jdhoffa — APPROVED** [review]
That is a bonkers amount of new files from `revdep`, but it all LGTM!

**Pull request merged**